src/github/orbit-helpers.js

```diff
@@ -64,7 +64,7 @@ export const orbitAPI = {
   async getMemberContributions(ORBIT_CREDENTIALS, username) {
     try {
       const response = await fetch(
-        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/find?github=${username}&api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
+        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/find?source=github&username=${username}&api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
```

Comment:
Unrelated to the issue at hand, but we could work on another update to move away from URL-based authentication and use the Authorization header instead

Reply:
I took a look at that as an option yesterday and ran into CORS issues when switching to Bearer-based authentication. Will investigate further.

Reply:
Hah—CORS issues, those are the worst. The following screenshot was taken when I was working on this extension:

![10 commit messages ranting about damn CORS](https://user-images.githubusercontent.com/2587348/120625304-db985f00-c461-11eb-8714-1e1ebab7f377.png)

Happy to take a look with you on this!